draft-ietf-tls-exported-authenticator.md

```diff
@@ -59,14 +58,15 @@ receiving peer.
 # Introduction
 
 This document provides a way to authenticate one party of a Transport Layer
-Security (TLS) connection to its peer using a Certificate message after the session
+Security (TLS) or Datagram Transport Layer Security (DTLS) connection to its peer
+using a Certificate message after the session
```

Comment:
This might change based on @kaduk's review (CertificateVerify instead of Certificate).